python-for-android/dists/kolibri/build.gradle

```diff
@@ -108,5 +108,6 @@ dependencies {
     implementation 'androidx.concurrent:concurrent-futures:1.1.0'
     implementation 'androidx.work:work-runtime:2.9.0'
     implementation 'androidx.work:work-multiprocess:2.9.0'
+    implementation "androidx.lifecycle:lifecycle-service:2.7.0"
```

Comment:
Did Android Studio warn you about this?

Reply:
It did not! :facepalm: Besides highlighting the inherited classes as red (missing), which still required you to click through and look at the class itself.